libs/product/guides/drivers.md

```diff
@@ -80,4 +80,9 @@ The only other thing you need to do is to make the product images in `@daffodil/
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## Dynamic driver switching
+
```

Comment:
I don't think we want to direct people to use the dynamic switch driver themselves. Instead I think we should use it to highlight how the drivers work.